R/PredictionSurv.R

```diff
@@ -13,12 +13,22 @@
 #' learner = lrn("surv.kaplan")
 #' p = learner$train(task, row_ids = 1:20)$predict(task, row_ids = 21:30)
 #' head(as.data.table(p))
+#' class(p$data$distr) # survival matrix stored
+#' p$distr # Matdist
```

Comment:
```suggestion
```

This is not always true (e.g., fully parametric distributions)

Reply:
It's true for the example though!

Reply:
But users aren't really meant to access the distribution via `$data$distr`, for them the main API is `$distr`. And by adding the comment it seems to imply that's *always* the case